extensions/wikia/CommunityHeader/CommunityHeaderController.class.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use \CommunityHeader\Sitename;
+use \CommunityHeader\Wordmark;
+use \CommunityHeader\Counter;
+use \CommunityHeader\WikiButtons;
+use \CommunityHeader\Navigation;
+
+class CommunityHeaderController extends WikiaController {
+
+	public function init() {
+	}
+
+	public function index() {
+		global $wgCityId;
+
+		$this->setVal( 'sitename', new Sitename() );
+		$this->setVal( 'wordmark', new Wordmark() );
+		$this->setVal( 'counter', new Counter() );
+		$this->setVal( 'wikiButtons', new WikiButtons() );
+		$this->setVal( 'navigation', new Navigation() );
+		$this->setVal(
+			'backgroundImageUrl',
+			( new SiteAttributeService() )
```

Comment:
to keep logic in one place, would it make sense to have getValue return empty string by default?